genai/api/npc_chat_api/README.md

```diff
@@ -0,0 +1,41 @@
+## NPC Chat API
+
+> [!NOTE]
+> Work in progress!
+
+The NPC Chat API demonstrates using [RAG](https://www.promptingguide.ai/techniques/rag) to create a 
+smart NPC for your game, enriched by world knowledge, NPC-specific knowledge, and secondhand
+knowledge from chat.
+
+To try it out:
+
```

Comment:
I'd make a note that you need to first set up the Quickstart cluster and run the `sed` command.

Reply:
I think it's safe to assume the reader has created the Quickstart cluster if they're in a deep directory here.

The `sed` thing will go away with https://github.com/googleforgames/GenAI-quickstart/pull/13